eg/Classes/PluginItem.py

```diff
@@ -78,6 +78,16 @@ def SearchFunc(obj):
             return False
         if not TreeItem.AskDelete(self):
             return False
+
+        if self.info.persistentData.config:
+            answer = eg.MessageBox(
+                eg.text.General.deleteConfigMessage,
```

Comment:
If i first got that dialog i was scared "what? delete all my config?! why?!". Then i realized it was just for the plugin. So it would be better to mention that:
`eg.text.General.deleteConfigMessage % self.info.name,`